src/raftstore/errors.rs

```diff
@@ -114,6 +114,9 @@ quick_error!{
             description("region is stale")
             display("StaleEpoch {}", msg)
         }
+        StaleCommand {
```

Comment:
I don't think backoff is necessary here, because the new leader should be elected, it can retry immediately.